sdk/keyvault/Azure.Security.KeyVault.Shared/src/ClientOptionsExtensions.cs

```diff
@@ -15,5 +16,13 @@ public static void ConfigureLogging(this ClientOptions clientOptions)
             allowedHeaderNames.Add("x-ms-keyvault-region");
             allowedHeaderNames.Add("x-ms-keyvault-service-version");
         }
+
+        public static void ConfigureLogging(this ClientPipelineOptions clientOptions)
```

Comment:
It would be ideal to find a way for these extensions to work with Azure.Core ClientOptions -- so the extension above I believe will be sufficient if we can prove the concept works for Azure clients as well as SCM-based clients.